Marlin/src/gcode/probe/G30.cpp

```diff
@@ -48,20 +52,38 @@ void GcodeSuite::G30() {
   const xy_pos_t pos = { parser.linearval('X', current_position.x + probe.offset_xy.x),
                          parser.linearval('Y', current_position.y + probe.offset_xy.y) };
 
-  if (!probe.can_reach(pos)) return;
-
+  if (!probe.can_reach(pos)) {
+    #if ENABLED(DWIN_LCD_PROUI)
+      PGM_P msg = {"Outside of Probing Area"};
+      SERIAL_ECHOLNPGM_P(msg);
+      ui.set_status(msg);
+    #endif
+    return;
+  }
   // Disable leveling so the planner won't mess with us
   TERN_(HAS_LEVELING, set_bed_leveling_enabled(false));
 
   remember_feedrate_scaling_off();
 
+  TERN_(DWIN_LCD_PROUI, gcode.process_subcommands_now(F("G28O"));)
```

Comment:
This gcode doesnt exist upstream here, was it intended to be submitted as well or is this line erroneous? Whats the definition of G280?

Reply:
It is G28 O, "O" means optional, if the axes are trusted then the printer ignores the G28 command.

Reply:
Scratch that lol read it as G280 (as in zero) on my tiny screen lol